adminSiteClient/ImagesIndexPage.tsx

```diff
@@ -88,40 +87,35 @@ function AltTextEditor({
     getAltText: ImageEditorApi["getAltText"]
 }) {
     const [value, setValue] = useState(text)
-    const [shouldAutosize, setShouldAutosize] = useState(false)
```

Comment:
I extracted this logic into a new component -`EditableTextarea` - which is used here in `ImagesIndexPage` and also `DodsIndexPage`